source/config/profileUpgradeSteps.py

```diff
@@ -376,3 +376,19 @@ def upgradeConfigFrom_10_to_11(profile: ConfigObj) -> None:
 			"hidBrailleStandard added to braille display auto detection excluded displays. "
 			f"List is now: {profile['braille']['auto']['excludedDisplays']}"
 		)
+
+
+def upgradeConfigFrom_11_to_12(profile: ConfigObj) -> None:
+	"""
+	use Screen layout where supported is disabled by default.

```

Comment:
Just for easier parsing

```suggestion
	"Use Screen layout where supported" is disabled by default.
```